modules/getting-started-with-cluster-configurations.adoc

```diff
@@ -0,0 +1,39 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/getting-started--with-openshift-gitops.adoc
+
+[id="getting-started-with-cluster-configuration_{context}"]
+= Getting started with cluster configuration
```

Comment:
IMO this section should be titled something like: Launching the operator provided Argo CD instance Can you check with Shiubhik and Dewan if that works.
The current title is so generic, cluster configuration for what? does not give the end user any clue about what this section is about. And I am not sure what cluster configs we talk about here.

Reply:
Changed the name so folks can see how it looks before making up their minds :)